seed/utils/search.py

```diff
@@ -6,9 +6,16 @@
 """
 import operator
 import re
```

Comment:
Are there supposed to be two different 'search.py' files?

Reply:
See above comment